extensions/wikia/DesignSystem/scripts/DesignSystemGlobalFooterTracking.js

```diff
@@ -0,0 +1,36 @@
+$(function ($) {
+	'use strict';
+	/**
+	 * Tracking helper function with most commonly used options. Overrides are passed in by callers.
+	 */
+	var track = Wikia.Tracker.buildTrackingFunction({
+		action: Wikia.Tracker.ACTIONS.CLICK,
+		category: 'footer',
+		trackingMethod: 'analytics'
+	});
+
+	/**
+	 * Parent click handler for events in the DS Global Footer
+	 * @param {object} event
+	 */
+	function clickTrackingHandler(event) {
+		var $element, label;
+
+		//Track only primary mouse button click
+		if (event.type === 'mousedown' && event.which !== 1) {
```

Comment:
What if user clicks the link with the middle mouse button to open the link in a different tab? Or right-click and then open/open in new tab? :|